### PR TITLE
Planet 5484 create zip on tag

### PR DESCRIPTION
**Pull request by Inwerpsel**
Ref: https://jira.greenpeace.org/browse/PLANET-5484

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -185,11 +185,24 @@ jobs:
...
+      - run: apk add curl zip git jq
+      - run: git submodule init && git submodule update
+      - run: npm ci
+      - run: npm run build
+      - run: rm -rf .circleci .git .githooks assets/src bin tests
+      - run: mkdir -p /tmp/workspace/
+      - run: zip -r /tmp/workspace/planet4-master-theme.zip . -x "node_modules/*"
+      - run: ls -lh /tmp/workspace/planet4-master-theme.zip
+      - run: curl https://api.github.com/repos/greenpeace/planet4-master-theme/releases/tags/${CIRCLE_TAG} | jq -r .upload_url | sed 's/{.*}/?name=planet4-master-theme/' > /tmp/workspace/upload_url
+      - run: cat /tmp/workspace/upload_url
```
Comment (comzeradd) [inline .circleci/config.yml:202]: If you want to make these steps less cryptic on CircleCI UI you can break them down to `name` and `command`.
https://circleci.com/docs/2.0/configuration-reference/#run

File: composer.json [hunk 2]
```diff
@@ -35,6 +35,9 @@
       "vendor/plugins/{$name}/": ["type:wordpress-plugin"]
     }
   },
+  "require": {
+    "inwerpsel/github-archive-installer": "^1.2.1"
```
Comment (comzeradd) [inline composer.json:39]: Can we bring this to `greenpeace` to give access to the whole team? in case more changes needed in the future.

Comment (Inwerpsel) [reply composer.json:39]: Of course, I just made the fork on my account because I wasn't sure if it would work :+1:

File: .circleci/config.yml [hunk 3]
```diff
@@ -185,11 +185,24 @@ jobs:
...
-      - image: rips/rips-cli:3
-    environment:
-      RIPS_APPLICATION_ID: 21
+      - image: mhart/alpine-node
     steps:
       - checkout
-      - run: rips-cli -vvv rips:scan:start -a $RIPS_APPLICATION_ID -p ~/project/ -t 1
+      - run: apk add curl zip git jq
+      - run: git submodule init && git submodule update
+      - run: PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm ci
```
Comment (comzeradd) [inline .circleci/config.yml:195]: :tada:

Comment (lithrel) [reply .circleci/config.yml:195]: waaait you can do that ? This thing kills me every time I reinstall dev env
mind blown

oooh there's even a package without chromium download https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteer-vs-puppeteer-core

Comment (Inwerpsel) [reply .circleci/config.yml:195]: Looks like Gutenberg in fact will use that instead, I think it's also introduced in the 5.5 release. https://github.com/WordPress/gutenberg/pull/20215

**Comment by Inwerpsel** [comment]
Looks like updating the lock file was breaking our php tests. This seems to be the lock file of an older install. Shouldn't be an issue as the lock file is only used in our CI for this repo.
https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/1803/workflows/8011defc-6247-43b1-af18-262304150296/jobs/9888/steps

File: .circleci/config.yml [hunk 4]
```diff
@@ -185,11 +185,39 @@ jobs:
...
     steps:
+      - run: apk add curl zip git openssh-client jq python make g++
       - checkout
-      - run: rips-cli -vvv rips:scan:start -a $RIPS_APPLICATION_ID -p ~/project/ -t 1
+      - run: git submodule init && git submodule update
+      - run: PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm ci
+      - run: npm run build
+      - run:
+          name: "Remove files we don't want in the zip file."
+          command: rm -rf .circleci .git .githooks assets/src bin tests
```
Comment (Inwerpsel) [inline .circleci/config.yml:199]: I included the ones of which I was sure they wouldn't be needed in the zip. Maybe there's some others which would make sense to exclude here? The total zip size is currently about 1.8MB.

**Review by comzeradd — APPROVED** [review]
:100:

**Pull request merged**